migrations/versions/720_add_framework_agreements_table.py

```diff
@@ -5,25 +5,29 @@
 Create Date: 2016-08-31 11:33:59.382347
 
 """
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
 
 # revision identifiers, used by Alembic.
 revision = '720'
 down_revision = '710'
 
-from alembic import op
-import sqlalchemy as sa
-from sqlalchemy.dialects import postgresql
 
 def upgrade():
-    op.create_table('framework_agreements',
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('supplier_id', sa.Integer(), nullable=False),
-    sa.Column('framework_id', sa.Integer(), nullable=False),
-    sa.Column('signed_agreement_returned_at', sa.DateTime(), nullable=True),
-    sa.Column('signed_agreement_details', postgresql.JSON(), nullable=True),
-    sa.Column('signed_agreement_path', sa.String(), nullable=True),
-    sa.ForeignKeyConstraint(['supplier_id', 'framework_id'], [u'supplier_frameworks.supplier_id', u'supplier_frameworks.framework_id'], ),
-    sa.PrimaryKeyConstraint('id')
+    op.create_table(
+        'framework_agreements',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('supplier_id', sa.Integer(), nullable=False),
+        sa.Column('framework_id', sa.Integer(), nullable=False),
+        sa.Column('signed_agreement_returned_at', sa.DateTime(), nullable=True),
+        sa.Column('signed_agreement_details', postgresql.JSON(), nullable=True),
+        sa.Column('signed_agreement_path', sa.String(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ['supplier_id', 'framework_id'],
+            [u'supplier_frameworks.supplier_id', u'supplier_frameworks.framework_id']
```

Comment:
We've lost a trailing comma here - it's *probably* entirely fine like this, but I'd be tempted to leave it in to preserve the historical record of what has actually been run.